org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterColorColor.java

```diff
@@ -475,51 +481,51 @@ public void updateConfiguration(@NonNull Configuration currentConfiguration,
     }
 
     @Override
-    public void attributeUpdated(ZclAttribute attribute) {
+    public void attributeUpdated(ZclAttribute attribute, Object val) {
         logger.debug("{}: ZigBee attribute reports {}", endpoint.getIeeeAddress(), attribute);
 
         synchronized (colorUpdateSync) {
             try {
                 if (attribute.getCluster().getId() == ZclOnOffCluster.CLUSTER_ID) {
                     if (attribute.getId() == ZclOnOffCluster.ATTR_ONOFF) {
-                        Boolean value = (Boolean) attribute.getLastValue();
+                        Boolean value = (Boolean) val;
```

Comment:
This seems inconsistent? Most other places where you get the value from the library there is an ```instanceof``` check. Why trust the type here and not everywhere else?